pkg/trisa/peers/peers.go

```diff
@@ -172,7 +188,8 @@ func (p *Peers) Lookup(commonName string) (peer *Peer, err error) {
 	if err = p.Add(info); err != nil {
 		return nil, err
 	}
-	return nil, nil
+
+	return peer, nil
```

Comment:
I think this was meant to return the peer here?

Reply:
Good catch!